drivers/modem/gsm_ppp.c

```diff
@@ -302,7 +304,18 @@ static void set_ppp_carrier_on(struct gsm_modem *gsm)
 	}
 
 	api = (const struct ppp_api *)ppp_dev->api;
-	api->start(ppp_dev);
+	if (is_booting_phase) {
+		api->start(ppp_dev);
+	} else {
+		int status;
+		struct net_if *network_int;
+		network_int = net_if_get_default();
+	/* Notify L2 to enable the interface */
```

Comment:
Please put empty line before the comment, and indent this line properly.

Reply:
addressed